### PR TITLE
Add idefics2 to the leaderboard

### PR DESCRIPTION
**Pull request by andimarafioti**
Hi 👋 

I added the code to run your evaluation with Idefics2 from Hugging Face 🤗 
I'm running this evaluation in our cluster and will submit the results to the leaderboard shortly, please let me know if you see any issues with the implementation!

### CODE REVIEW

File: vlms/idefics2/idefics2_vqa_updbench.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                ],
+            }]
+
+            text = processor.apply_chat_template(messages, add_generation_prompt=True)
+
+            inputs = processor(images=image, text=text, return_tensors="pt").to('cuda')
+
+            with torch.inference_mode():
+                generated_text = model.generate(**inputs, max_new_tokens=500)
+                outputs = processor.batch_decode(generated_text, skip_special_tokens=True)[0]
```
Comment (AtsuMiyai) [inline vlms/idefics2/idefics2_vqa_updbench.py:110]: Could you remove the input text from the output sentences? In the current version, this `outputs` has the input sentences.

Current outputs: 
> User: Which one is the correct caption of this image?
> A. A chocolate cake with icing next to plates and spoons.
> B. Stuffed teddy bear sitting next to garbage can on the side of the road.
> C. A group of baseball players playing a game of baseball.
> D. Two stainless steel sinks with mirrors and a fire extinguisher.
>  
> Assistant: D. Two stainless steel sinks with mirrors and a fire extinguisher.

Ideal outputs: 

> D. Two stainless steel sinks with mirrors and a fire extinguisher.

**Review by AtsuMiyai — CHANGES_REQUESTED** [review]
Thanks for your PR! 

Could you remove the input text from the output sentences in `vlms/idefics2/idefics2_vqa_updbench.py`? In the current version, this `outputs` has the input sentences.

Other points are LGTM for me!

**Comment by andimarafioti** [comment]
I addressed the changes and I'm re-running the processing + eval to see if there are any changes in performance.

**Comment by AtsuMiyai** [comment]
@andimarafioti 
Thanks for your modification! 
The current version is good for merging. Thanks for your contribution!

**Review by AtsuMiyai — APPROVED** [review]

**Pull request merged**